content/en/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -53,6 +53,8 @@ list) but you cannot add a new finalizer. You also cannot modify the `deletionTi
 object once it is set.
 
 * After the deletion is requested, you can not resurrect this object. The only way is to delete it and make a new similar object.
+
+* Service accounts and service account tokens bound to a pod become invalid 1 minute after their `.metadata.deletionTimestamp`, regardless of finalizers.
```

Comment:
I don't think this is a great spot to put this doc. There's lots of object types and various treatment of deletionTimestamps. This clarification could go in the service account token documentation.

Reply:
I moved it to `### Assign a ServiceAccount to a Pod {#assign-to-pod}`

Reply:
Thanks, but this is still the wrong place. We're explaining the concept of what a finalizer is, in general.

Reply:
@sftim what do you mean for 'in general'?

Reply:
Look at this search:
https://www.google.com/search?q=ServiceAccount+token+k8s

the pages at the top of the results list are good candidates for where to add this detail.

Reply:
Hi @sftim can you check if it fits better now?